pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -256,17 +256,16 @@ sub _centos_symlink_protection {
         );
     }
     if ( !($ruid) && !($rack911) && !($bluehost_ea3) && !($bluehost_ea4) && !_has_kc_free_patch_set($kernelcare_state) ) {
+        my $doclink = $self->_lh->maketext(
+            q{You do not appear to have any symlink protection enabled through a properly patched kernel on this server, which provides additional protections beyond those solutions employed in userland. Please review [output,url,_1,the documentation,_2,_3] to learn how to apply this protection.},
+            ($is_ea4) ? 'https://go.cpanel.net/EA4Symlink' : 'https://go.cpanel.net/apachesymlink', 'target', '_blank'
+        );
         $security_advisor_obj->add_advice(
             {
                 'key'        => 'Apache_no_symlink_protection',
                 'type'       => $bad,
-                'text'       => $self->_lh->maketext('No symlink protection detected'),
-                'suggestion' => $self->_lh->maketext(
-                    'You do not appear to have any symlink protection enabled on this server. You can protect against this in multiple ways. Please review the following [output,url,_1,documentation,_2,_3] to find a solution that is suited to your needs.',
-                    ($is_ea4) ? 'https://go.cpanel.net/EA4Symlink' : 'https://go.cpanel.net/apachesymlink',
-                    'target',
-                    '_blank'
-                ),
+                'text'       => q{Kernel does not support the prevention of symlink ownership attacks.},
+                'suggestion' => $doclink,
```

Comment:
I don't have a strong opinion either way, but is there a reason that you split this out above?  This is different from what we do elsewhere and it does make review a bit more difficult.

Reply:
No reason, it was a game time decision. I see it's inconsistent, but if we're not going to localize this change isn't worth our time if we're not going localize the string above. Please consider accepting this PR as is.

Reply:
As I said, I don't have a strong opinion, so you needn't fix this issue if you don't want to.